src_erl/Communication_Layer/http_Nerlserver/src/nerl_tools.erl

```diff
@@ -0,0 +1,129 @@
+-module(nerl_tools).
+
+-include("nerl_tools.hrl").
+
+-export([setup_logger/1]).
+-export([start_connection/1, http_request/4, getHostPort/5, getShortPath/3]).
+-export([string_to_list_int/1, deleteOldJson/1]).
+-export([multipart/2, read_all_data/2]).
+-export([getdeviceIP/0]).
+
+setup_logger(Module) ->
+  logger:add_handler(Module, Module, #{}), 
```

Comment:
This is the configuration I used for single line logger format, please use it also for unified format: 
   
logger:set_handler_config(default, formatter, {logger_formatter, #{}}),
logger:set_module_level(Module, all).


Consider unify this module with nerl.erl from erlbridge.